options/pin.go

```diff
@@ -1,13 +1,23 @@
 package options
 
+import "fmt"
+
 type PinAddSettings struct {
 	Recursive bool
 }
 
+type TypeSettings struct {
+	Type string
+}
```

Comment:
This does not seem used, but I'm not sure

Reply:
removed